src/libraries/System.Linq.Expressions/src/CompatibilitySuppressions.xml

```diff
@@ -139,6 +139,48 @@
     <Left>ref/net8.0/System.Linq.Expressions.dll</Left>
     <Right>lib/net8.0/System.Linq.Expressions.dll</Right>
   </Suppression>
+  <Suppression>
```

Comment:
These changes should be reverted

Reply:
The build fails without these two changes.